source/content/guides/wordpress-composer/02-wordpress-composer-managed.md

```diff
@@ -0,0 +1,111 @@
+---
+title: WordPress with Composer on Pantheon
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock and Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith, jspellman814, jazzsequence]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/wordpress-composer-managed
+anchorid: wordpress-composer-managed
+---
+
+<Alert title="Early Access" type="info" icon="leaf">
+
+The WordPress Composer Managed upstream is available for Early Access participants. Features for WordPress Composer Managed are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. To learn how you can enroll in our Early Access program, visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K).
```

Comment:
Can we link to a definition of Early Access?

Reply:
Not sure that we can link text in an Alert, but I'll test it out.

Reply:
@CdrMarks is this the page you had in mind to link to?

Reply:
@jspellman814 I was thinking https://pantheon.io/docs/oss-support-levels#early-access since this is where the repository badges link.